CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
   - `extra-doc-files` requires setting `cabal-version` to at least
     1.18; this is now done properly.
   - Accept bool for `condition` (see #230)
+  - Allow local files to be used as defaults (#248)
```

Comment:
Please also add `(fixes #248)` to the commit message, so that GitHub will close the issue automatically once this goes on `master`.